e2e-tests/cypress/support/insurance/account/verify-account-email.js

```diff
@@ -0,0 +1,33 @@
+import { INSURANCE_ROUTES as ROUTES } from '../../../../constants/routes/insurance';
+
+const {
+  ACCOUNT: { CREATE: { VERIFY_EMAIL }, SIGN_IN },
+} = ROUTES;
+
+const exporterEmail = Cypress.env('GOV_NOTIFY_EMAIL_RECIPIENT');
+
+/**
+ * verifyAccountEmail
+ * Get the exporter (with verification hash) directly from the API,
+ * Mimic "cliking email verification link" in an email inbox by manually navigating to the URL
+ */
+const verifyAccountEmail = () => {
+  // get exporter
+  cy.getExporterByEmail(exporterEmail).then((response) => {
+    const { data } = response.body;
```

Comment:
Should we consider `response` and `data` void check, otherwise please wrap the in try-catch.

Reply:
response/data check is handled by `cy.getExporterByEmail` so no need for it here as well - i've added try/catch to both.